src/app/api/v1/user/email/route.ts

```diff
@@ -0,0 +1,142 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { getToken } from "next-auth/jwt";
+import { NextRequest, NextResponse } from "next/server";
+import AppConstants from "../../../../../appConstants";
+
+import {
+  getSubscriberByEmail,
+  deleteResolutionsWithEmail,
+} from "../../../../../db/tables/subscribers";
+import {
+  addSubscriberUnverifiedEmailHash,
+  removeOneSecondaryEmail,
+  getEmailById,
+} from "../../../../../db/tables/emailAddresses.js";
+
+import { sendVerificationEmail } from "../../../utils/email";
+
+import { validateEmailAddress } from "../../../../../utils/emailAddress";
+import { getL10n } from "../../../../functions/server/l10n";
+const l10n = getL10n();
```

Comment:
`getL10n` processes the request headers to determine which locale to load a string from, so it can't be called from the top level - it needs to be inside a request context, e.g. inside `POST()`.

From there, you can pass it on to function that need it - I added `getStringLookup` that takes it as an argument and then returns a replacement for `getMessage`, to ease the transition.